product/roadmap.md

```diff
@@ -50,10 +53,50 @@ We are dedicated to maintain support for three different Kubernetes versions wit
 - Provide disaster recovery procedures for Otomi core applications, such as Gitea, Keycloak and Harbor
 - Show a compliance report (code vulnerabilities, image vulnerabilities, security violations, container vulnerabilities, config audit, exposed secrets) per workload
 
-## After Q2 2024
+### After Q2 2024
 
 After Q2 2024 we have the following goals:
 
 - Make Otomi more plugable by enabling users to bring their own platform apps
 - Migrate to ambient mesh using eBPF
 - Remove platform app forms in favor of generated values that can be customized in an editor
+
+## Removed features by release
+
+### v3.0
+
+#### Drone
+
+Otomi will stop delivering Drone as a platform app.
+The Tekton app is introduced in Otomi v2.0 as a replacement. There is no migration procedure, Drone is used primarely as an Otomi deployment pipeline. However if you used it as for custom pipeline then you may need to migrate them Tekton.
```

Comment:
`it as for custom pipeline` -> `it for custom pipelines`

Reply:
ok